server/api/order.js

```diff
@@ -1,64 +1,144 @@
 const router = require('express').Router()
-const {Order, OrderProduct} = require('../db/models')
+const {Order, OrderProduct, Product} = require('../db/models')
 
+//GET all orders and their associated product and orderProduct info
+router.get('/', async (req, res, next) => {
+  try {
+    const orders = await Order.findAll({
+      include: {
+        model: Product,
+        through: OrderProduct 
+      }
+    })
+    res.json(orders)
+  } catch (err) { next (err) }
+
+})
+
+//POST new order for session user
 router.post('/', async (req, res, next) => {
   try {
-    const newOrder = await Order.create({ userId: req.user.id })
+    const order = await Order.create({ 
+        userId: req.user.id,
+        status:false
+      })
+    res.json(order)
+  } catch (err) {
+    next(err)
+  }
+})
+
+//GET all unbought products and their associated orderProduct info
```

Comment:
Are the routes below not using req.user to account for guests?

Reply:
yes! I couldn't think of any other way to consistently match guest orders with their respective guests other than to save the orderId in the front end and local storage -- borrowing that from my previous cart pitch. Otherwise there's just no way to retrieve that information for them like we could with users.